core-ui/src/components/ApiRules/ApiRuleForm/AccessStrategyForm/AccessStrategyForm.js

```diff
@@ -60,34 +83,12 @@ export default function AccessStrategyForm({
               />
             </FormItem>
             <FormItem>
-              <FormSelect
-                defaultValue={selectedType}
-                aria-label="Access strategy type"
-                id="select-1"
-                onChange={e => {
-                  const newStrategy = {
-                    ...strategy,
-                    accessStrategies: [
-                      {
-                        ...strategy.accessStrategies[0],
-                        handler: e.target.value,
-                      },
-                    ],
-                  };
-                  // strategy type changed, reset current values
-                  if (e.target.value !== strategy.accessStrategies[0].handler) {
-                    newStrategy.accessStrategies[0].config = {};
-                  }
-                  setStrategy(newStrategy);
-                  handleFormChanged();
-                }}
-              >
-                {Object.values(accessStrategyTypes).map(ac => (
-                  <option key={ac.value} value={ac.value}>
-                    {ac.displayName}
-                  </option>
-                ))}
-              </FormSelect>
+              <Dropdown
```

Comment:
This dropdown on apirules page is lower than others elements in the row (have margin-top:4px)